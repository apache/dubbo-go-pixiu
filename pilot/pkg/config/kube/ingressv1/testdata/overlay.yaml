apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: foo
  namespace: ns
spec:
  rules:
    - host: foohost.bar.com
      http:
        paths:
          - backend:
              service:
                name: service1
                port:
                  number: 4200
---
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: foo2
  namespace: ns
spec:
  rules:
    - host: foohost.bar.com
      http:
        paths:
          - path: /second
            backend:
              service:
                name: service2
                port:
                  number: 4200