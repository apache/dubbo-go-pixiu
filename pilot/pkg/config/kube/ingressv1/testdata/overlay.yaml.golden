apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  creationTimestamp: null
  name: foo-istio-autogenerated-k8s-ingress-ns
  namespace: istio-system
spec:
  selector:
    istio: ingressgateway
  servers:
  - hosts:
    - '*'
    port:
      name: http-80-ingress-foo-ns
      number: 80
      protocol: HTTP
---
apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  creationTimestamp: null
  name: foo2-istio-autogenerated-k8s-ingress-ns
  namespace: istio-system
spec:
  selector:
    istio: ingressgateway
  servers:
  - hosts:
    - '*'
    port:
      name: http-80-ingress-foo2-ns
      number: 80
      protocol: HTTP
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/route-semantics: ingress
  creationTimestamp: null
  name: foohost-bar-com-foo-istio-autogenerated-k8s-ingress
  namespace: ns
spec:
  gateways:
  - istio-system/foo-istio-autogenerated-k8s-ingress-ns
  hosts:
  - foohost.bar.com
  http:
  - match:
    - uri:
        exact: /second
    route:
    - destination:
        host: service2.ns.svc.mydomain
        port:
          number: 4200
      weight: 100
  - match:
    - {}
    route:
    - destination:
        host: service1.ns.svc.mydomain
        port:
          number: 4200
      weight: 100
---
